generics/constants/common.js

```diff
@@ -56,5 +56,8 @@ module.exports = {
   "CREATED_BY_ME" : "createdByMe",
   "DEFAULT_PAGE_SIZE" : 100,
   "DEFAULT_PAGE_NO" : 1,
-  "DHITI" : "dhiti"
+  "DHITI" : "dhiti",
+  "CREATE_OBSERVATION": "/create-observation/",
+  "TAKE_SURVEY" : "/take-survey/",
```

Comment:
@priyanka-TL  for survey should not we have create-survey instead of take-survey ?? Same like observations and projects.  Just a suggestions  :+1: